WordPress/src/main/java/org/wordpress/android/ui/main/SitePickerAdapter.java

```diff
@@ -387,6 +388,7 @@ void setEnableEditMode(boolean enable) {
 
         mIsMultiSelectEnabled = enable;
         mSelectedPositions.clear();
+        notifyDataSetChanged();
```

Comment:
A `notifyDataSetChanged()` call is made in `LoadSitesTask.onPostExecute` from `loadSites()` just below this line.   We can remove this since it's redundant.